CountItems/CountMaster.js

```diff
@@ -1,11 +1,14 @@
 const async = require('async');
 const { reshapeExceptionError } = require('arsenal').errorUtils;
+const { threshold, frequency } = require('../utils/constants');
 
 class CountMaster {
     constructor(params) {
         this.log = params.log;
         this.manager = params.manager;
         this.client = params.client;
+        this.metrics = params.metrics;
+        CountMaster.waitingForPromScraping = false;
```

Comment:
why do we need to put this here?
this like all this would fit very nicey in masterProcess directly, without splitting the implementation in 3 places:
* start() is called by masterProcess, it can also start the metrics server (can even be done immediately)
* start's `callback` is provided by masterProcess, so it can extend it to handle the termination

Reply:
done here : https://github.com/scality/s3utils/pull/312/commits/be0f3e16b518690540561a94a498ff07dc3860b0#diff-46cbf734fd7b9022c7bbf597a8a5deb96829b581ca817a861bbc3bf1e86274a9R52-R82